interfaces/builtin/fwupd.go

```diff
@@ -111,6 +225,56 @@ const fwupdPermanentSlotAppArmor = `
   dbus (bind)
       bus=system
       name="org.freedesktop.fwupd",
+
+  # fwupdtool may need to stop fwupd
+  dbus (send)
+      bus=system
+      interface=org.freedesktop.DBus.Properties
+      path=/org/freedesktop/systemd1
+      member=Get{,All}
+      peer=(label=unconfined),
+
+  dbus (send)
+      bus=system
+      interface=org.freedesktop.DBus.Properties
+      path=/org/freedesktop/systemd1/unit/*
+      member=GetAll
+      peer=(label=unconfined),
+
+  dbus (send)
+      bus=system
+      path=/org/freedesktop/systemd1
+      interface=org.freedesktop.systemd1.Manager
+      member=GetUnit
+      peer=(label=unconfined),
+
+  dbus (send)
+      bus=system
+      interface=org.freedesktop.systemd1.Unit
+      path=/org/freedesktop/systemd1/unit/*
```

Comment:
`unit/*` seems overly broad - the comment above says that fwupdtool may need to stop fwupd - can we make this be say:

```
path=/org/freedesktop/systemd1/unit/fwupd*
```

instead?

Reply:
Tested and changed.